core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay

```diff
@@ -1,22 +0,0 @@
-registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-dispatcher quay.io/openshift-knative/knative-eventing-kafka-broker-dispatcher
```

Comment:
There is a bug in the generator that ends up dropping this files that are needed

Reply:
Ah ok. I was wondering, but didn't remember related issues last time... Updated it...